v3/src/components/graph/hooks/use-plot.ts

```diff
@@ -55,37 +56,9 @@ export const usePlotResponders = (props: IPlotResponderProps) => {
     instanceId = useInstanceIdContext(),
     refreshPointPositionsRef = useCurrent(refreshPointPositions)
```

Comment:
You probably don't need the ref any more, i.e. you should be able to just pass `refreshPointPositions` directly to `useDebouncedCallback()`.